packages/eslint-plugin-pf-codemods/lib/rules/v5/fileUploadField-cb-param-updates.js

```diff
@@ -0,0 +1,14 @@
+const { addCallbackParam } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/1234
+module.exports = {
+  meta: { fixable: "code" },
+  create: addCallbackParam(["FileUploadField"], {
+    onChange: {
+      defaultParamName: "_event",
+      previousParamIndex: 2,
+      otherMatchers: /^_?(ev\w*|e$)/,
+    },
+    onTextChange: '_event'
+  }),
```

Comment:
```suggestion
  create: addCallbackParam(["FileUploadField"], {
    onTextChange: '_event'
  }),
```

onChange was removed from FileUploadField, so we probably don't need to supply a fix for a prop that will just get removed.